src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -27,6 +29,28 @@ const MainLayout = async (props: Props) => {
     return <SignInButton autoSignIn />;
   }
 
+  const accountId = session?.user?.subscriber?.fxa_uid;
+
+  let userId;
+  if (accountId) {
```

Comment:
I'm assuming `accountId` can't be `0`, but to be sure, it's probably better to do something like:

```suggestion
  if (typeof accountId === "number") {
```

(Or a `string` if that's what it is, of course.)